### PR TITLE
[#92502564] Add GCE configs

### PR DESCRIPTION
**Pull request by dcarley**
This is a mirror of AWS.

There are a few things outstanding, but this branch has already dragged on a long time, accumulated a lot of commits, and needs constant rebasing. So I'm going to raise this for review now and we can fix the other things within the same story.
- API isn't available over SSL because we only have access to L4 load balancing and not L7 to terminate SSL
- I haven't been able to verify whether the router is working after making the health check changes. Might require more tweaks to firewalls

I have however been able to use the outputted IPs to provision the cluster with Ansible, login, add a platform and deploy an app.

### CODE REVIEW

File: aws/outputs.tf [hunk 1]
```diff
@@ -6,6 +6,10 @@ output "docker.private.ip" {
   value = "${aws_instance.tsuru-docker.private_ip}"
 }
 
+output "docker-registry.private_ip" {
```
Comment (mikkoc) [inline aws/outputs.tf:9]: Did this slip in or was it intended?

Comment (dcarley) [inline aws/outputs.tf:9]: The commit f318c0f231086dc7e122ff504607c71dc9fca7b9 that it belongs to explains why.

**Comment by mikkoc** [comment]
If it's too much time to change that, we can merge and tackle it with a separate PR! :+1:

**Comment by annashipman** [comment]
I agree that the comment raised in a07cf55f should go on our tech debt list. As to how we'd fix it or whether we'd get around to it, we don't have to decide that now, and if we're lucky it will get fixed by someone else :smile:.

Is the issue raised in 8d33f46d worth adding to tech debt also?

Other than that, :+1: for getting rid of long-running branches and the other issues that you and @mikkoc raise can be addressed in subsequent PRs.

What I'd like to see in the future is some shared code and the differences in a different place, as already this has diverged from AWS in quite a few very small ways. However, that is a story for another day.

I'm going to merge this now but let's agree what to do about the tech debt stories.

**Comment by mikkoc** [comment]
Well done! :+1:

**Comment by dcarley** [comment]
Thanks! Still not quite there, but a lot closer.

**Pull request merged**